lang.json

```
@@ -1,28 +1,33 @@
 {
 	"languages": {
 		"en": "English",
-		"ja": "Japanese"
+		"ja": "Japanese",
+		"pt": "Portuguese"
 	},
 	"general": {
 		"learnMore": {
 			"description": "Used in links such as in code setting descriptions",
 			"en": "Learn more",
-			"ja": "もっと詳しく知る"
+			"ja": "もっと詳しく知る",
+			"pt": "Saiba mais"
 		},
 		"by": {
 			"description": "Used in phrases like '[Sketch Title] by [User name]'",
 			"en": "by",
-			"ja": "本"
+			"ja": "本",
+			"pt": "por"
 		},
 		"saveAsFork": {
 			"description": "navigation - when sketch is changed",
 			"en": "Save as Fork",
-			"ja": "スケッチ"
+			"ja": "スケッチ",
+			"pt": "Salvar bifurcação"
```

```suggestion
			"pt": "Salvar Bifurcação"
```
Capitalization of the second word?